YAXLib/YAXSerializer.cs

```diff
@@ -28,6 +28,9 @@ namespace YAXLib
     /// </summary>
     public class YAXSerializer
     {
+
+        public static CultureInfo CurrentCulture { get; set; } = CultureInfo.InvariantCulture;

```

Comment:
Can you change it to `CultureInfo.CurrentCulture` to keep it backward compatible?